test/unittests/VisitTest.php

```diff
@@ -1,106 +0,0 @@
-<?php
```

Comment:
Why does this PR remove this other test?

Reply:
moving back